aws/resource_aws_instance_test.go

```diff
@@ -1123,6 +1123,53 @@ func TestAccAWSInstance_volumeTagsComputed(t *testing.T) {
 	})
 }
 
+func TestAccAWSInstance_blockDeviceVolumeTags(t *testing.T) {
```

Comment:
I think we need another test for this sort of config:

```hcl
resource "aws_instance" "tst" {
  ami                         = var.ami
  instance_type               = "t2.micro"

  root_block_device {
    delete_on_termination = true
    volume_size           = "10"
    volume_type           = "standard"

    tags = {
      Name = %q
    }
  }
}

resource "aws_volume_attachment" "tst" {
  device_name = "/dev/sdf"
  volume_id   = aws_ebs_volume.tst.id
  instance_id = aws_instance.tst.id
}

resource "aws_ebs_volume" "tst" {
  availability_zone = var.az
  size              = 1
  type              = "gp2"

  tags = {
    Name = %q
  }
}
```

Reply:
So, we have `TestAccAWSInstance_volumeTagsWithAttachedVolume` for this. 
Should I modify it somehow?